### PR TITLE
feat: SAML2 Documentation

### PR DESCRIPTION
**Pull request by DeadSuperHero**
TODO
- [x] Adjust SSO doc to point to SAML2 page
- [x] Add Screenshots
- [x] Include 1-2 more Troubleshooting questions
- [x] Request Final Copy Review

### CODE REVIEW

**Comment by DeadSuperHero** [comment]
@jessla This is now ready for review!

File: src/collections/_documentation/accounts/saml2.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+---
+
+Sentry provides a generic auth provider for [SAML2 based authentication](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language), which allows Owners of a Sentry organization to manually configure any SAML2-enabled IdP system. Documented below are the general steps for integration.
+
+{% capture __alert_content -%}
+SAML2 SSO requires an Enterprise Plan.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
```
Comment (maheskett) [inline src/collections/_documentation/accounts/saml2.md]: Should this be `level="warning"`?

File: src/collections/_documentation/accounts/saml2.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+* Identity Provider initiated SLO (Single Logout)
+
+## 1. Register Sentry with IdP
+Before connecting Sentry to the Identity Provider (IdP), it’s important to first register Sentry as an application on the IdP’s side. Sentry’s SAML endpoints are as follows, where the `{organization_slug}` is substituted for your organization slug:
+
+<table class="table"><tbody valign="top"><tr><th>ACS:</th><td><code class="docutils literal">https://sentry.io/saml/acs/{organization_slug}/</code></td></tr><tr><th>SLS:</th><td><code class="docutils literal">https://sentry.io/saml/SLS/{organization_slug}/</code></td></tr><tr><th>Metadata:</th><td><code class="docutils literal">https://sentry.io/saml/metadata/{organization_slug}/</code></td></tr></tbody></table>
+
+**What are these three things?**
+* `ACS` means *Assertion Consumer Service*, and is used for establishing a session based on rules made between your IdP and the service provider it is integrating with. Please note: Sentry’s ACS endpoint uses HTTP-POST bindings
+* `SLS` stands for *Single Logout Service*, and is used to address logout requests from the IdP.
```
Comment (maheskett) [inline src/collections/_documentation/accounts/saml2.md]: We've been trying to italicize notes:

_Note: Sentry’s ACS endpoint uses HTTP-POST bindings `SLS` stands for *Single Logout Service*, and is used to address logout requests from the IdP._

File: src/collections/_documentation/accounts/saml2.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+## 2. Register IdP with Sentry
+There are three distinct methods for registering your IdP with Sentry.io: Metadata, XML, and data fields. Each method is broken down below, and will produce the same end result.
+
+### Using Metadata URL
+This method only requires a Metadata URL provided by the IdP platform. After it has been supplied, Sentry will automatically fetch the metadata it needs.
+
+{% asset saml2-metadata-url.png %}
+
+### Using Provider XML
+For this method to work, an administrator needs to provide the contents of the IdP’s generated metadata file. Once the contents are pasted directly into the text field, Sentry will do the rest. Note that Sentry.io does not require a signing certificate.
```
Comment (maheskett) [inline src/collections/_documentation/accounts/saml2.md]: This note as well

File: src/collections/_documentation/accounts/saml2.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+{% asset saml2-provider-xml.png %}
+
+Here’s an example of what the Metadata XML contents look like.
+
+```
+
```
Comment (maheskett) [inline src/collections/_documentation/accounts/saml2.md]: Do you need the space here?

File: src/collections/_documentation/accounts/saml2.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+* `SingleSignOnService`
+
+In this example, the `SingleLogoutService` isn’t provided by the IdP, and is treated as an optional value within Sentry.
+
+
+## 3. Map IdP Attributes
+
+{% include components/alert.html
+  title="Note on field names"
+  content="Metadata field names can vary from one provider to another. For example, Microsoft Azure AD refers to these very metadata fields as `Claims`, while Okta refers to them as `Attributes`. Similarly, one platform might use `user.email`, while another vendor uses `emailaddress`."
```
Comment (maheskett) [inline src/collections/_documentation/accounts/saml2.md]: IMO the pink on red is a little hard to read. What do you think about bolding these terms instead? **Claims**, **Attributes**, etc.

File: src/collections/_documentation/accounts/saml2.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+Sentry makes use of Just-In-Time provisioning for member accounts; any new member created within an Identity Provider will have an account automatically created in Sentry when that member attempts to sign into Sentry for the first time.  
+
+**What happens to Sentry.io accounts that exist prior to SAML being enabled/enforced?**  
+Existing members will receive an email notifying them of the new SAML authentication option once it is turned on (regardless of enforcement) and they'll be able to link the accounts in the IdP system with their Sentry memberships.  
+
+**Does Sentry deprovision members if they are no longer present in the Identity Provider?**  
+At this time, Sentry's SAML2 integration does not automatically deprovision inactive user accounts.
+Instead, the member remains inside of Sentry.io without any means to log in, as they can no longer access the IdP platform for sign-on. For now, inactive member accounts will need to be removed manually by a Manager or an Owner in Sentry.
+
+**Attempting to set up SAML2 SSO with an IdP results in a failure with the message “The provider did not return a valid user identity” What is happening here?**
```
Comment (maheskett) [inline src/collections/_documentation/accounts/saml2.md]: Should there be a period after "valid user identity"?

File: src/collections/_documentation/accounts/sso.md [hunk 7]
```diff
@@ -91,3 +68,7 @@ As part of the Auth0 SSO configuration, you must provide the Auth0 Identity Prov
 In your Rippling admin dashboard locate the Sentry app in the list of suggested apps and select it.
 
 When prompted with the Rippling Metadata URL, copy this into the Sentry Rippling provider configuration. You will have to complete the Rippling application configuration before completing the sentry provider configuration.
+
+#### Custom SAML2 Integration
+
+For other SAML2 SSO providers not listed above,Sentry provides a generic connectors for [SAML2 based authentication]({%- link _documentation/accounts/saml2.md -%}), which may be configured manually.
```
Comment (maheskett) [inline src/collections/_documentation/accounts/sso.md]: Should this be:

For other SAML2 SSO providers not listed above, Sentry provides generic connectors for [SAML2 based authentication]({%- link _documentation/accounts/saml2.md -%}), which may be configured manually.

**Pull request merged**